media-gfx/netgen/netgen-6.2.2204.ebuild

```diff
@@ -0,0 +1,152 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_10 )
```

Comment:
Is it compatible with python3_11?

Reply:
IIRC during the update to this version on ::sci, I tested it and it wasn't.
I can check again, if you insist, else I would leave this to a future update.

Reply:
> IIRC during the update to this version on ::sci, I tested it and it wasn't. I can check again, if you insist, else I would leave this to a future update.

Ah yes, you are right, I forgot about that. Lets leave it as is for now and re-check when there is a new release. Python3.11 is preliminary scheduled to become the default around June so we still have plenty of time.